packages/sdk/package-lock.json

```diff
@@ -7791,8 +7791,7 @@
       "dependencies": {
         "ansi-regex": {
           "version": "5.0.0",
-          "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-5.0.0.tgz",
-          "integrity": "sha512-bY6fj56OUQ0hU1KjFNDQuJFezqKdrAyFdIevADiqrWHwSlbmBNMHp5ak2f40Pm8JTFyM2mqxkG6ngkHO11f/lg==",
+          "resolved": "",
```

Comment:
Is this blank field expected?

Reply:
this modify is done by npm audit.
I think once the lock file and package.json need change in repo, it's better to run audit locally.
thanks!